BioArchLinux/r-arrow/PKGBUILD

```diff
@@ -32,16 +35,17 @@ optdepends=(
   r-hms
   r-knitr
   r-lubridate
+  r-pillar
   r-pkgload
   r-reticulate
   r-rmarkdown
   r-stringi
   r-stringr
+  r-sys
   r-testthat
   r-tibble
+  r-tzdb
   r-withr
-  libcurl-compat
-  openssl
```

Comment:
@starsareintherose  why does this pkg optionally depends on `libcurl-compat` and `openssl`?

Reply:
I forget, you can check

Reply:
> I forget, you can check

I cannot check, see https://github.com/BioArchLinux/Packages/commit/608ef961a51c71a6ca1451738e230ee46a5f23fa